internal/ssh/server.go

```
@@ -69,11 +72,39 @@ func newServer(ca string) *ssh.Server {
 		cmd.Dir = user.HomeDir
 
 		execCmd(s, cmd, uid, gid)
-
 	})
 
+	key, err := rsa.GenerateKey(rand.Reader, 2048)
+	if err != nil {
+		log.Printf("could not generate rsa key: %s", err)
+		return nil
```

just curious, shouldn't we return the error in this function?

---

Changed it to a fatal error.